src/jsx/intrinsic-elements.ts

```diff
@@ -199,7 +202,7 @@ export namespace JSX {
     | 'strict-origin-when-cross-origin'
     | 'unsafe-url'
 
-  type HTMLAttributeAnchorTarget = '_self' | '_blank' | '_parent' | '_top' | string
+  type HTMLAttributeAnchorTarget = StringLiteralUnion<'_self' | '_blank' | '_parent' | '_top'>
```

Comment:
Which do you like better this or the above:

```ts
  type HTMLAttributeAnchorTarget = '_self' | '_blank' | '_parent' | '_top'
//...
  StringLiteralUnion<HTMLAttributeAnchorTarget>
```

I reffered to https://github.com/DefinitelyTyped/DefinitelyTyped/commit/462531abb774dd76995632249db601002a33e93d .